### DISCUSSION TITLE
Group options defined in main interface vs. Options -> Application Groups

### DISCUSSION THREAD

**Discussion by Forever-A-Hermit** [tnodir/fort#59] [Q&A] [answered]
Are these different settings? Which should I be using to limit bandwidth per application/application group? If I change one, the changes are not reflected in the other.

![Screenshot 2023-03-24 140605](https://user-images.githubusercontent.com/15148336/227607975-e3f7a649-1338-4967-813b-ea87a485e351.png)

![Screenshot 2023-03-25 114542](https://user-images.githubusercontent.com/15148336/227728021-698b3eb9-3626-4c2a-9490-8d4d101fb7e8.png)

**Comment by tnodir** [comment]
Let me show how to limit bandwidth of application:

1. Create an Application Group and set its "Download/Upload speed limit":
![fort-appgroup](https://user-images.githubusercontent.com/303655/227727923-1703417c-08aa-4bcb-9d32-9b3c85978884.jpg)

2. Set the Application Group to desired program:
![fort-program](https://user-images.githubusercontent.com/303655/227728011-0bdc97fe-ddd5-4358-a771-b65e1d4bf92a.jpg)

**Comment by tnodir** [comment, marked as answer]
> Are these different settings?

1. D/U speed limit is defined only per App Group
2. Several programs can be added to App Group
3. Programs can be defined in the Programs window or App Group's settings
4. Wildcard in program path can be used only in App Group's settings
5. Program in the Programs window overrides App Group's settings (if it defined in both places)
